variables.tf

```diff
@@ -117,3 +117,7 @@ variable "nat_gateway_enabled" {
   default     = "true"
 }
 
+variable "assign_public_ip_on_launch" {
+  description = "Flag to enable/disable whether instances launched into public subnets should be assigned a public IP address"
+  default     = "false"
```

Comment:
@nadenf please update to use `type = bool` and update the default to use the bool `false` instead of the string `"false"`?